.github/workflows/tests.yml

```diff
@@ -88,6 +88,41 @@ jobs:
           TESTING_FORMULAE: ${{ steps.formulae-detect.outputs.formulae_to_fetch }}
         run: brew test-bot --only-bottles-fetch --testing-formulae="$TESTING_FORMULAE"
 
+  report_analytics:
+    runs-on: ubuntu-latest
+    needs: formulae_detect
+    steps:
+      - name: Publish Analytics to Summary
+        env:
+          TESTING_FORMULAE: ${{needs.formulae_detect.outputs.testing_formulae}}
+        run: |
+          if [[ -z "$TESTING_FORMULAE" ]]
+          then
+              exit 0
+          fi
+
+          {
```

Comment:
I think this could probably all be replaced with `brew info --analytics $TESTING_FORMULAE`?

Reply:
I tried, but that immediately puts it in tables that aren't as much more verbose than this one. And `brew info --json` doesn't seem to have that data.